package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "my-project",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

[suggestion] add start script to make it easier to start the server instead of needing to drill down into the server directory.  

OR more index.js to the root of the repo to make it easier to start the server. 
```suggestion
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "start": "node start/server/index.js"
  },
```